src/test/scala/parsley/CharTests.scala

```diff
@@ -1,3 +1,5 @@
+package parsley
```

Comment:
Dotty got stricter about referring to private constructors (e.g., Success) from the default package. Seems... good?

Reply:
yeah, I'm surprised I missed this honestly